packages/components/src/modal/DebouncedModal.test.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { DEFAULT_DEBOUNCE_MS } from '@deephaven/react-hooks';
+import { act, render, screen } from '@testing-library/react';
+import DebouncedModal from './DebouncedModal';
+import Modal from './Modal';
+
+const mockChildText = 'Mock Child';
+const children = (
+  <Modal>
+    <div>{mockChildText}</div>
+  </Modal>
+);
+
+beforeAll(() => {
+  jest.useFakeTimers();
+});
+
+afterAll(() => {
+  jest.useRealTimers();
+});
+
+describe('display modal after debounce', () => {
+  it('should render the modal after the debounce time has passed', () => {
+    const { rerender } = render(
+      <DebouncedModal isOpen={false}>{children}</DebouncedModal>
+    );
+    expect(screen.queryByTestId('debounced-modal-backdrop')).toBeNull();
+    expect(screen.queryByText(mockChildText)).toBeNull();
+
+    act(() => {
+      rerender(<DebouncedModal isOpen>{children}</DebouncedModal>);
+    });
+    expect(screen.queryByTestId('debounced-modal-backdrop')).not.toBeNull();
+    expect(screen.queryByText(mockChildText)).toBeNull();
```

Comment:
`.toBeInTheDocument()` or w/e that matcher is called. Most expect calls in these tests could swap to that matcher

Reply:
Always forget about that one.